python/pyspark/ml/evaluation.py

```diff
@@ -545,6 +545,99 @@ def getDistanceMeasure(self):
         return self.getOrDefault(self.distanceMeasure)
 
 
+@inherit_doc
+class RankingEvaluator(JavaEvaluator, HasLabelCol, HasPredictionCol,
```

Comment:
Can we set "since 3.0.0" here or is that not the python convention?

Reply:
I reviewed other several python files and found that no python classset since annotation here.
It seems that it's the python convention.